b2b-app/imports/ui/admin/register/step-body.js

```diff
@@ -0,0 +1,155 @@
+import React, { useContext, useEffect } from 'react'
+import { Typography, Paper } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import StepContent from '@material-ui/core/StepContent'
+
+import { meteorCall } from '/imports/ui/utils/meteor'
+import { RegisterContext } from './context'
+import Form from './form'
+import {
+  termsFormBridge,
+  contactFormBridge,
+  emergencyFormBridge,
+  aboutFormBridge,
+  avatarFormBridge,
+} from './form/form-schemas'
+import ConfirmData from './form/confirm-data'
+import ConditionalWrap from './conditional-wrap'
+
+const useStyles = makeStyles((theme) => ({
+  formHeading: {
+    marginBottom: theme.spacing(2),
+    textAlign: 'center',
+  },
+  paper: {
+    margin: theme.spacing(4, 0),
+    padding: theme.spacing(4, 2),
+  },
+  formWrap: {
+    maxWidth: 600,
+    margin: '0 auto',
+  },
+  frontMatter: {
+    marginBottom: theme.spacing(4),
+  },
+}))
+
+const schemaBridges = [
+  contactFormBridge,
+  aboutFormBridge,
+  emergencyFormBridge,
+  avatarFormBridge,
+  termsFormBridge,
+]
+
+function getStepHeading(stepIndex) {
+  switch (stepIndex) {
+    case 0:
+      return "Let's start with some basic contact details"
+    case 1:
+      return "Let's learn a bit about you"
+    case 2:
+      return 'Who should we contact in an emergency?'
+    case 3:
+      return 'Choose your avatar'
+    case 4:
+      return "Just making sure everything's correct"
+    default:
+      return 'Unknown stepIndex'
+  }
+}
+
+const StepBody = (props) => {
+  const classes = useStyles()
+  const {
+    steps,
+    activeStep,
+    models,
+    isEditingStep,
+    isSubmitting,
+    dispatch,
+    isMobile,
+  } = useContext(RegisterContext)
+
+  useEffect(() => {
+    const submitData = async () => {
+      if (!isSubmitting) return
+      console.log('sending to server')
+
+      const s = await meteorCall(
+        'insert.registrations',
+        'saving',
+        Object.assign({}, ...models)
+      )
```

Comment:
are we sure this is working? they dont seem to add to the registrations page on my end?

Reply:
they seem to submit but not display, not exactly sure why yet

Reply:
Ha ha, time to write some tests to exercise the code :)

Reply:
Works for me :P Tested in desktop just then tho. Might be a bug in mobile?

Reply:
Not a mobile bug its an Andrew bug, I just didnt realise that the search term persisted from our last meeting, hence nothing was updating. All good now, it was working the entire time 😅